src/App.tsx

```diff
@@ -10,9 +11,15 @@ export const App = () => {
 
       <div className="section">
         <div className="container">
-          <h1 className="title">Home Page</h1>
-          <h1 className="title">Page not found</h1>
-          <PeoplePage />
+          <Routes>
+            <Route path="/" element={<h1 className="title">Home Page</h1>} />
+            <Route path="/" element={<Navigate to="/" />} />
```

Comment:
```suggestion
            <Route path="/home" element={<Navigate to="/" />} />
```
that is the reason tests fail
![image](https://github.com/mate-academy/react_people-table-advanced/assets/86923089/1b3c71de-1602-440c-bc30-c0f1dc7ca6fb)